src/setup/cortx_deploy

```diff
@@ -50,9 +50,9 @@ function conf_get {
 function apply_config {
     for file in "$SOLUTION_CONFIG"/*
     do
-        if [ -f "$file" ]
+        [ -f "$file" ] &&
+        if [[ "$(conf_get $(yaml_config $file) 'cluster')" != "null" || "$(conf_get $(yaml_config $file) 'cortx')" != "null" ]]
```

Comment:
What is yaml_config ? If its name of the function then its incorrect. Function names should typically be verb e.g. parse_url if you really meant that.

Reply:
Renamed yaml_config() to make_url()